packages/shared_preferences/shared_preferences_tool/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: shared_preferences_tool
 description: "DevTools extension for package:shared_preferences. Manage SharedPreferences efficiently. Edit, search, and view keys."
 publish_to: 'none'
 
-version: 1.0.0+1
+version: 1.0.0+2
```

Comment:
This should be 1.0.1; we don't use `+` versions beyond `0.x`. (This was apparently not caught in the initial review, but we should fix it now.)